ReactAndroid/src/main/res/devsupport/layout/dev_loading_view.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<RelativeLayout
```

Comment:
This RelativeLayout is unused. The height can be applied on the TextView. You can change this to a <merge>